docs/developer-docs/backend/motoko/access-control.mdx

```diff
@@ -30,10 +33,61 @@ Open a terminal window on your local computer, if you don’t already have one o
 
 Create a new project by running the following command:
 
+<Tabs groupId="version">
```

Comment:
suggestion: if you find yourself repeating the same snippet, it could be turned into a React component and imported into a `.mdx` file.

Reply:
Updated to use the React component for all 'hello_world' examples